components/ai_chat/resources/untrusted_conversation_frame/untrusted_conversation_frame.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang="en" dir="$i18n{textdirection}">
+  <head>
+    <meta charset="UTF-8">
+    <title>$i18n{siteTitle}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="//resources/brave/css/reset.css">
+    <link rel="stylesheet" href="//resources/css/text_defaults.css">
+    <link rel="stylesheet" href="//resources/brave/fonts/manrope.css">
+    <link rel="stylesheet" href="//resources/brave/fonts/poppins.css">
+    <link rel="stylesheet" href="//resources/brave/fonts/inter.css">
+    <script src="//resources/js/load_time_data_deprecated.js"></script>
+    <script src="/strings.js"></script>
+    <style>
```

Comment:
nit: can we load `variables.css` in the head here, rather than importing it in JS?